app/controllers/ProfileViewsStreamController.scala

```diff
@@ -1,27 +1,25 @@
 package controllers
 
-import play.api.mvc.{Action, Controller}
 import data.ServiceClient
-import ui.{HtmlStream, Pagelet}
-import ui.HtmlStreamImplicits._
 import play.api.libs.concurrent.Execution.Implicits._
+import ui.HtmlStreamImplicits._
+import ui.{HtmlStream, Pagelet}
 
 /**
  * An example standalone module that represents the "Who's Viewed Your Profile" module from the LinkedIn homepage.
- * It's identical to the Wvyp controller, except that the data is streamed to the browser as soon as it's available in
+ * It's identical to the ProfileViewsController controller, except that the data is streamed to the browser as soon as it's available in
  * small "pagelets", similar to Facebook's BigPipe.
  */
-object WvypStream extends Controller {
+object ProfileViewsStreamController {
+
+  def stream = HtmlStream {
```

Comment:
If it's not an `Action`, the class does not extend `Controller`, and it doesn't show up in `routes`, it is not really a controller. That defeats the whole "standalone endpoint" idea.